utils/GRI_bed_filter/version1_gene-only/gri_hg38_v1.2.R

```diff
@@ -0,0 +1,167 @@
+#GRI.V1=protein_coding gene + HGMD mutations (50 flanking bp) + Cinvar mutations (50 flanking bp)
```

Comment:
Wouldn't it be better to merge both hg19 and hg38 into one script?

Reply:
Yes